projects/kubernetes/kubernetes/build/run_tests.sh

```diff
@@ -44,10 +54,10 @@ export PATH="${GOPATH}/bin:${MAKE_ROOT}/kubernetes/third_party/etcd:${PATH}"
 MAX_RETRIES=5
 # There are flakes in upstream tests, the process also caches passing results
 # so on rerun it only runs the tests which failed
-for i in $(seq 1 $MAX_RETRIES); 
-do 
+for i in $(seq 1 $MAX_RETRIES);
+do
   [ $i -gt 1 ] && sleep 5
-  make test && ret=0 && break || ret=$?; 
+  make test && ret=0 && break || ret=$?;
```

Comment:
IntelliJ reformatted this on its own. This change does not matter for the 1.24 release